tornjak-frontend/src/components/tornjak-api-helpers.js

```diff
@@ -252,6 +254,39 @@ class TornjakApi extends Component {
       })
   }
 
+  // clusterDelete - returns success message after successful deletion of a cluster in manager mode
+  async clusterDelete(serverName, inputData, clustersListUpdateFunc, globalClustersList) {
```

Comment:
should return boolean instead, since it kind of is checking for api result which is not defined as a const anywhere and may change in the future.